apps/src/templates/lessonOverview/activities/Activity.jsx

```diff
@@ -0,0 +1,31 @@
+import PropTypes from 'prop-types';
+import React, {Component} from 'react';
+import ActivitySection from '@cdo/apps/templates/lessonOverview/activities/ActivitySection';
+import color from '@cdo/apps/util/color';
+
+const styles = {
+  activityHeader: {
+    color: color.purple
+  }
+};
+
+export default class Activity extends Component {
+  static propTypes = {
+    activity: PropTypes.object
+  };
+
+  render() {
+    const {activity} = this.props;
+
+    return (
+      <div>
+        <h2 style={styles.activityHeader}>{`${activity.displayName} (${
+          activity.time
+        } minutes)`}</h2>
```

Comment:
I think we will need to translate these, so please extract a translatable string here

Reply:
Good call. I translated everything that was on the lesson plan page so far as well.